### PR TITLE
Update Engage project profile

### LINKED ISSUE #1031: Update Project Profile: Engage
### Overview
We want the Engage profile card to be up to date so that new volunteers can have current information when they come to the website.

### Project card
_insert url of their card here before sending - https://www.hackforla.org/projects/engage
https://github.com/hackforla/website/blob/gh-pages/_projects/engage.md

### Action Items
- [x] Update Enage project card using the information from this issue https://github.com/hackforla/product-management/issues/36
- [x] Please confirm user acceptance by asking PM: Bryon to review #364 
- [x] Once accepted remove dependency on https://github.com/hackforla/product-management/issues/36 .

### PR DESCRIPTION
**Pull request by glenflorendo**
- Update project details on Engage project page
- Update meeting times in VRMS
- Import two new images to replace photo in the project page and the hero image in the Engage project page

Resolves #1031
See also #1033

Screenshot_2021-05-22 Engage by Hack for LA
Screenshot_2021-05-22 Hack for LA

### CODE REVIEW

File: _projects/engage.md [hunk 1]
```diff
@@ -1,48 +1,65 @@
...
 identification: '138791772'
 title: Engage
-description: Engage is a civic participation platform. Currently in beta, Engage makes it easier for residents of Santa Monica, CA to offer their feedback on policy issues that City Council is considering. Engage aims to increase access for community stakeholders who are unable to attend public meetings or may otherwise feel unheard by their local government.
-image: /assets/images/projects/engage.jpg
-alt: 'Santa Monica city council closed in session on CityTV channel.'
-image-hero: /assets/images/projects/engage-hero.jpg
+description: 'Engage facilitates and streamlines the preparation of meeting agendas, getting word out to the local residents and compiling feedback from the public.
+Engage is a platform to help connect residents (and other stakeholders) to local government. Instead of requiring constituents to be present at meetings, which presents a host of challenges: work schedules, childcare, accessibility, fear that their voice would not be heard, or not understanding the meeting agenda.'
+image: /assets/images/projects/engage-logo.png
+alt: 'Engage logo.'
```
Comment (qiqicodes) [inline _projects/engage.md:7]: Is the illustration also part of the Engage logo? if not, we should elaborate on the alt text for this image.

File: _projects/engage.md [hunk 2]
```diff
@@ -1,48 +1,65 @@
...
+  - Node.js
   - PostgreSQL
+  - MongoDB
 location:
-  # - Santa Monica
   - Remote
 tools: Hotjar, Google Analytics
-program area: 'Citizen Engagement'
-partner: City of Santa Monica
+vertical: 'Citizen Engagement'
```
Comment (qiqicodes) [inline _projects/engage.md]: a recent Merged PR changed the `vertical` to `program area`, please change accordingly.

File: _projects/engage.md [hunk 3]
```diff
@@ -1,48 +1,65 @@
...
-image: /assets/images/projects/engage.jpg
-alt: 'Santa Monica city council closed in session on CityTV channel.'
-image-hero: /assets/images/projects/engage-hero.jpg
+description: 'Engage facilitates and streamlines the preparation of meeting agendas, getting word out to the local residents and compiling feedback from the public.
+Engage is a platform to help connect residents (and other stakeholders) to local government. Instead of requiring constituents to be present at meetings, which presents a host of challenges: work schedules, childcare, accessibility, fear that their voice would not be heard, or not understanding the meeting agenda.'
+image: /assets/images/projects/engage-logo.png
+alt: 'Engage logo.'
+image-hero: /assets/images/projects/engage-hero-2.png
 alt-hero: 'Neon sign for the Santa Monica pier. Palm trees and sunset in the background.'
```
Comment (qiqicodes) [inline _projects/engage.md]: Is the illustration also part of the Engage logo? if not, we should elaborate on the alt text for this image.

**Review by qiqicodes — CHANGES_REQUESTED** [review]
Please do the following:
- [x] edit the alt text for engage-hero-2.png
- [x] change the  `vertical` to `program area`.

**Comment by glenflorendo** [comment]
On hold. Pending alternative hero image from Engage team.

**Review by jbubar — COMMENTED** [review]
Hey Glen, So sorry about the insane delay on this issue. 
everything looks good. The Test Site, and Site links are broken... but this is not your fault.
I think we should merge this. Ill write a message on the original issue asking for an updated site link

**Comment by glenflorendo** [comment]
> Hey Glen, So sorry about the insane delay on this issue.
> everything looks good. The Test Site, and Site links are broken... but this is not your fault.
> I think we should merge this. Ill write a message on the original issue asking for an updated site link

Hey @jbubar . Thanks for reviewing! I think the links are working, but there's something going on with the certs with the sites. I think you can bypass the screen by going through `Advanced` and continuing from there.

**Pull request merged**